packages/docs/components/TableOfContents/LayoutUtils.tsx

```diff
@@ -10,6 +10,10 @@ export const TableOfContents: React.FC = () => {
           <strong>measureText()</strong>
           <div>Get dimensions of text</div>
         </TOCItem>
+        <TOCItem link="/docs/layout-utils/fill-text-box">
+          <strong>fillTextBox()</strong>
+          <div>Calculate text is fill the box</div>
```

Comment:
This seems grammatically not correct

Reply:
I can't think of a simple function description. 
Can you provide some suggestions?